Check this beautiful code

DsArmor opened the pull request.

DsArmor committed:
merge and resolve conflict in api.yml

DsArmor committed:
add create and get default methods

DsArmor committed:
correct request body in api.yml

DsArmor committed:
final connection with user and meeting

dvalkoff commented on src/main/java/ru/valkov/calendarapp/exceptions/ExceptionWrapper.java:
```diff
@@ -8,21 +8,27 @@
 
 public class ExceptionWrapper {
     public static <T, R> ResponseEntity<R> wrap(Function<T, R> function, T arg) {
+        System.out.println("There post");
```
sout'ы убрать надо, наверн
я ща логирование добавлю нормальное

dvalkoff commented on src/main/java/ru/valkov/calendarapp/meeting/MeetingMapper.java:
```diff
@@ -0,0 +1,40 @@
+                .beginDateTime(request.getBeginDateTime().toLocalDateTime())
+                .endDateTime(request.getEndDateTime().toLocalDateTime())
+                .location(request.getDescription())
+                .owner(user) //todo add owner
```
todo тож убрать можно тут

dvalkoff commented on src/main/java/ru/valkov/calendarapp/user/UserMapper.java:
```diff
@@ -25,4 +25,15 @@ public UserResponse map(User user) {
+                .firstName(response.getFirstName())
+                .middleName(response.getMiddleName())
+                .lastName(response.getLastName( ))
+                .status(UserStatus.ONLINE) // todo change to normal
```
тут тож убрать можно

DsArmor replied:
почему убрать тут ( ты где то записал, потому что можем забыть

dvalkoff replied:
а, это я писал? ну задачку добавили на доску
так что можно забить

dvalkoff commented on src/main/java/ru/valkov/calendarapp/meeting/MeetingMapper.java:
```diff
@@ -0,0 +1,40 @@
+public class MeetingMapper {
+
+    @Autowired
+    private UserMapper userMapper;
```
тут можно final сделать и @RequiredArgsConstructor аннотацию добавить, чтоб как во всем проекте

DsArmor committed:
fixed and change field ddl-auto to connect it to docker-compose field

- чтобы не дропалась база при запуске не через докер сервера измени себе в env cofiguration DDL_AUTO на update
